### ISSUE TITLE
[Bug]: Confirmation on delete project covers required input

### ISSUE THREAD

**Issue by mffap** [zitadel/zitadel#5107]
### Preflight Checklist

- [X] I could not find a solution in the documentation, the existing issues or discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Environment

ZITADEL Cloud

### Describe the bug

When deleting a project, I need to enter the project's name. The name is covered by the modal.

### To reproduce

1. Create a project with a name that you for sure can't remember
2. Click the bin icon
3. Enter the project name to delete

### Screenshots

<img width="602" alt="Screenshot 2023-01-26 at 09 18 58" src="https://user-images.githubusercontent.com/1366906/214790852-48ee4ce4-f82d-44b3-b975-3914e479e324.png">

### Expected behavior

Show the project's name in the modal so it can be copied / typed into the field for confirmation.

### Version

2.18.0

**Comment by hifabienne** [comment]
@juergrinaldi From a ux perspective is it better to ask for something related to the object or just to type "delete"?

**Comment by juergrinaldi** [comment]
That depends on how hard you wanna interupt the deletion process (to avoid accidental deletes). There are several approaches to this with different pros and cons (sorted by hardness of interrupt:

- type the name of the object (or something similar relevant). this is the most conscious as it hardly interrupts the process and makes a semantic connection to the object. drawback is that this can be VERY cumbersome when the names of the objects are complex or long or when the user has to delete several objects in a row (users for example). It can also be a blocker if the object name allows special characters (i.e.: é,à, Ð,ð,...) but the admin does not have the keyboard layout to type it.
- type 'Delete' or a random word. This still interrupts the deletion workflow enough to get confirmation but gets rid of the problem with complex/long names and special characters. Con: It loses the semantic connection (or lower is with 'Delete'). Note: The random word is an option to overcome an automatic behavior if the user knows the word beforehand after several execution of the process)
- Ask the user to confirm the Deletion by pressing another button (are you sure you want to delete XY?) or click a checkbox. This interrupts the workflow a little bit, but can lead to an automatic behavior that contradicts that.
- Don't ask at all. This is the most liberate approach and the easiest for the user. On destructive workflows like deletion this is only recommended if the user can undo this afterwards.

**TL;DR:** I'd go with the **second option** (type DELETE) on a non-reversable process, as this is imo the best of both worlds. (gets rid of the problem with the special characters and complexity but still interrupts a deletion process quite severely)

**Comment by juergrinaldi** [comment]
Two more things:

- We might want to prevent Copy/Paste on that particular input field if we go with one of the first two options. Otherwise users might circumnavigate the interruption and it is obsolete.
- I'd also advice against the first suggestion if there's an option for deleting multiple objects at once. (which name should be typed?)

**Comment by hifabienne** [comment]
I have created a new issue accroding to this, because we want to change for typing "delete"
And this should not only apply for project, but also for orgs, users and apps.
Will close this therefore, please subscribe here: https://github.com/zitadel/zitadel/issues/5131
